cfht2caom2/cfht_name.py

```diff
@@ -75,12 +75,10 @@
 from cfht2caom2.metadata import Inst
 
 
-__all__ = ['CFHTName', 'COLLECTION', 'ARCHIVE']
+__all__ = ['CFHTName']
 
 
-COLLECTION = 'CFHT'
-ARCHIVE = 'CFHT'
-
+# TODO - is this still required?
```

Comment:
It sounds like `archive` is a super set of `collection` so we might need both?

Reply:
COLLECTION/ARCHIVE has been moved to a value that's retrieved from the configuration in `config.yml`, which is why these values have been removed from the module.

I'm generally not sure what to do about the bigger question of which is the more accurate name.